wiki/Article_styling_criteria/Writing/en.md

```diff
@@ -0,0 +1,312 @@
+# Writing
+
+*For the formatting standards, see: [Article style criteria/Formatting](../Formatting)*
+
+All English articles must use plain English.
+
+## Language register
+
+There are two language registers on the wiki: neutral and informal.
+
+The neutral register is the default on the wiki.
+
+The informal register is special and given to a small handful of articles. Some examples with this register include [FAQ](/wiki/FAQ) and [Help Centre](/wiki/Help_Centre).
+
+For both registers, all rules in this part of the article style criteria must be followed. Exceptions for either register will be noted.
+
+## British English
+
+British English spelling variants must be used when there are conflicting spellings of English words. For example, use:
+
+- `colour` instead of `color`
+- `centre` instead of `center`
+- `skilful` instead of `skillful`
+- `analyse` instead of `analyze`
+
+## Capitalisation
+
+### Article names
+
+Article names, when written in a paragraph, must not be capitalised, unless it is a link that links to the article or it is a proper noun (e.g. BanchoBot). For example:
+
+```markdown
+The [Chat Console](/wiki/Chat_Console) is where you go to chat, using IRC (Internet Relay Chat). The chat console is located in the game client.
+```
+
+### Game modifiers
+
+Game modifiers must have the word `mod` after the mod name to reduce ambiguity. These must be spelt as seen below (letter casing and spacing must match):
+
+- `Easy` or `EZ`
+- `No Fail` or `NF`
+- `Half Time` or `HT`
+- `Daycore` or `DC`
+- `Hard Rock` or `HR`
+- `Sudden Death` or `SD`
+- `Perfect` or `PF`
+- `Double Time` or `DT`
+- `Nightcore` or `NC`
+- `Fade In` or `FI`
+- `Hidden` or `HD`
+- `Flashlight` or `FL`
+- `Relax` or `RL`
+- `Autopilot` or `AP`
+- `Target Practice` or `TP`
+- `Spun Out` or `SO`
+- `1K`, `2K`, `3K`, `4K`, `5K`, `6K`, `7K`, `8K`, and `9K`
+  - If referring to the key mods collectively, use `xK`.
+- `Co-op` or `CO`
+- `Random` or `RD`
+- `Auto` or `AT`
+- `Cinema` or `CM`
+- `Touch Device` or `TD`
```

Comment:
AFAIK, Touch Device is not a mod

Reply:
This list is also missing some mods:
- ScoreV2
- Fade Out
- No Video
- 10K

Reply:
![image](https://user-images.githubusercontent.com/14264992/87100013-e8d8a980-c210-11ea-8706-24ac0baf88cc.png)

Touch Device is referring to that icon on the right. It's on the website only and I assume this is automatically applied to scores that used a touch device during play, but I'm not for sure.

---

Those other mods that you listed, except for ScoreV2, are no longer mods. I'm not sure if it would be practical to list them here. Then again, touch device is also a weird one to be on here as well...

Reply:
It'd be nice to list the correct spelling and casing bc they are stubs in the Game Modifiers rewrite

Reply:
Alright, sounds good.

Touch Device stub might be worth doing.

Reply:
May want to mention the Mirror mod for mania